temporal_capi/src/plain_date.rs

```diff
@@ -155,6 +155,10 @@ pub mod ffi {
                 .map_err(Into::into)
         }
 
+        pub fn equals(&self, other: &Self) -> bool {
```

Comment:
thought: I don't actually think these methods are necessary, but I guess it doesn't harm to have them since we're trying to match the API